Fix an unhandled rejection in the sdk realtime client when the connection closed during a heartbeat ping (#27846)
directus/directus: .changeset/tall-clouds-cheer.md

---
'@directus/sdk': patch
---

> Fixed an unhandled promise rejection in the realtime client when the connection closed while a heartbeat ping was being handled

Reviewer: ```suggestion
Fixed an unhandled rejection in the sdk realtime client when the connection closed during a heartbeat ping
```
